Mu2eG4/inc/Mu2eG4MTRunManager.hh

```diff
@@ -52,6 +53,7 @@ namespace mu2e {
     Mu2eG4MTRunManager& operator=( Mu2eG4MTRunManager const & ) = delete;
 
     Mu2eG4Config::Top conf_;
+    Mu2eG4ResourceLimits mu2elimits_;
```

Comment:
I am curious why you make a copy of this.  It's a small object so it's not very important.